sdcm/utils/hdrhistogram.py

```diff
@@ -312,12 +327,13 @@ def _get_workload_type_by_hdr_tag(self, hdr_tag):
         #    Examples: 'fn--write', 'fn--write-batch', 'fn--get', 'fn--get-many', 'fn--read'.
         # 3) 'scylla-bench' has identical tag names for reads and writes - 'co-fixed' and 'raw'.
         #    It doesn't have 'mixed' workload type, so it's mode should be used for detecting the tag data type.
-        # 4) NOT_SUPPORTED: 'ycsb', it supports HDR histograms, but doesn't use tags in it.
-        #    So, the 'ycsb' case should be handled separately.
+        # 4) 'ycsb', it supports HDR histograms, but doesn't use tags in it.
```

Comment:
Please, add info about tag usage like it is done for the first 3 points above.
Does it follow one of above ways? Does it have 4th unique approach?
It will be useful for anyone later trying to understand/debug ycsb-based test runs.